addons/website/__manifest__.py

```diff
@@ -300,55 +300,58 @@
     },
     'new_page_templates': {
         'basic': {
-            'blank': ['s_text_block_title_blank'],
+            'blank': ['s_text_block_h1'],
+            '1': ['s_text_block_h1', 's_text_block', 's_image_text', 's_text_image'],
+            '2': ['s_text_block_h1', 's_picture', 's_text_block'],
+            '3': ['s_parallax', 's_text_block_h1', 's_text_block', 's_three_columns'],
+            '4': ['s_text_cover'],
+            '5': ['s_text_block_h1', 's_text_block', 's_features', 's_quotes_carousel'],
+            '6': ['s_text_block_h1', 's_table_of_content'],
         },
         'about': {
-            '0': ['s_banner', 's_text_block', 's_three_columns', 's_company_team', 's_quotes_carousel'],
-            '1': ['s_picture', 's_showcase', 's_three_columns', 's_company_team', 's_quotes_carousel'],
-            '2': ['s_banner', 's_three_columns', 's_showcase', 's_company_team', 's_quotes_carousel'],
-            '3': ['s_picture', 's_three_columns', 's_company_team', 's_quotes_carousel', 's_showcase'],
-            '4': ['s_banner', 's_text_block', 's_three_columns', 's_quotes_carousel', 's_showcase'],
-            '5': ['s_picture', 's_three_columns', 's_showcase', 's_quotes_carousel', 's_company_team'],
+            'full': ['s_text_block_h1', 's_image_text', 's_text_image', 's_numbers', 's_picture', 's_quotes_carousel', 's_references'],
+            'full_1': ['s_text_block_h1', 's_three_columns', 's_text_block_h2', 's_company_team', 's_references', 's_quotes_carousel', 's_call_to_action'],
+            'mini': ['s_cover', 's_text_block_h2', 's_text_block_2nd', 's_picture_only', 's_text_block_h2_contact', 's_website_form'],
+            'personal': ['s_text_cover', 's_image_text', 's_text_block_h2', 's_numbers', 's_features', 's_call_to_action_about'],
+            'map': ['s_text_block_h1', 's_text_block', 's_numbers', 's_text_image', 's_text_block_h2', 's_text_block_2nd', 's_map', 's_images_wall'],
+            'timeline': ['s_banner', 's_text_block_h2', 's_text_block', 's_timeline', 's_call_to_action_about'],
```

Comment:
Initially it was all numbers and using a blocks table available in the task description - but when running through designers it was decided to create templates that address specific use cases instead. I personally would rather have had significant names for every pages. Note that updating these is very annoying because the names appear through all themes. (It could be done though)

Reply:
No it's very fine :+1: 
Just got curious as why they named those but not other, but well it's just keys